pom.xml

```diff
@@ -25,6 +25,7 @@
         <integration-tests.skip>true</integration-tests.skip>
         <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
         <jaxb-maven.version>2.3.1</jaxb-maven.version>
+        <spring.security.oauth.version>2.1.0.RELEASE</spring.security.oauth.version>
```

Comment:
If this is using spring boot, do we need to manually declare this version?  Isn't there an oauth-boot-starter or something?